### PR TITLE
Header is fixed when scrolling inside a pane

### PR DESCRIPTION
**Pull request by umbe1987**
This would tackle the feature discussed in #141 .
If the pane has a very olng content which needs to be scrolled, now the header is not scrolled and stays fixed, while the rest scrolls (both vertically and horizontally).
Updated the example "ol3.html".
https://rawgit.com/umbe1987/sidebar-v2/sidebar_scroll/examples/ol3.html

### CODE REVIEW

**Comment by Turbo87** [comment]
this is a breaking change, increases the filesize and might not be needed by everyone. it also only changes the ol3 implementation but none of the others. I'd prefer to keep the code simple and not include this. those users that want it should be able to copy it from this PR.

thanks for your contribution though 🙏

**Comment by umbe1987** [comment]
ok no problem. Thanks again for this project!

**Comment by tstibbs** [comment]
@Turbo87 Just wondering, if the change was made consistently across all implementations, are you open to it? Or do you think the current behaviour is what most people are looking for?

Just that I'm seeing the same problem as @umbe1987 - where you scroll to the end of a long tab only to find that the 'close' button has disappeared. It does seem to me that in most cases you'd want to fix the header, but maybe there's a usecase I'm missing?

**Comment by Turbo87** [comment]
I guess it might make sense as a default, but it might be easier to adjust the `sidebar-header` class to use something like `position: absolute/fixed`. would have to experiment with it to figure out what the best solution is, but I'd like to avoid the extra wrapper element.

File: css/ol3-sidebar.css [hunk 1]
```diff
@@ -210,3 +208,18 @@
   .sidebar-right.collapsed ~ .sidebar-map .ol-attribution,
   .sidebar-right.collapsed ~ .sidebar-map .ol-full-screen {
     margin-right: 52px; } }
+
+.sidebar-inner {
+  position: absolute;
+  top: 40px;
```
Comment (albfan) [inline css/ol3-sidebar.css]: I'd add `width:100%`

**Pull request closed without merging**